docker/sandbox-bundled/manifests/complete-agent.yaml

```
@@ -1744,7 +1744,7 @@ spec:
           value: minio
         - name: FLYTE_AWS_SECRET_ACCESS_KEY
           value: miniostorage
-        image: ghcr.io/flyteorg/flyteagent:1.9.1
+        image: ghcr.io/flyteorg/flyteagent:1.10.0
```

```suggestion
        image: ghcr.io/flyteorg/flyteagent:1.9.1
```
This is my mistake, it should be 1.9.1